src/plugins/vis_types/xy/public/editor/components/options/metrics_axes/label_options.tsx

```diff
@@ -34,8 +34,8 @@ function LabelOptions({
 }: LabelOptionsProps) {
   const setAxisLabelRotate = useCallback(
     (paramName: 'rotate', value: Labels['rotate']) => {
-      // @ts-expect-error ts upgrade v4.7.4
-      setAxisLabel(paramName, Number(value));
+      const rotation = Number(value) as LabelRotation;
+      setAxisLabel(paramName, rotation);
```

Comment:
I think you could just pass `value` directly:
```suggestion
      setAxisLabel(paramName, value);
```

Reply:
There is a covcertion to Number that I dont want to change because it might create a bug.